pywinauto/application.py

```diff
@@ -846,7 +845,7 @@ def connect(self, **kwargs):
             try:
                 timeout = float(kwargs['timeout'])
                 del kwargs['timeout']
-                wait_until_passes(self.timeout, Timings.app_connect_retry, partial(self.Connect, **kwargs), (ValueError))
+                time.sleep(timeout)

```

Comment:
@vasily-v-ryabov ,apparently, we have a different understanding of the timeout. 
tell me, please, how do I then check that timeut expired?
it is impossible to simulate this situation. in order to display the exception need at least a connection failure